pkg/manifests/manifests_test.go

```diff
@@ -1344,6 +1344,48 @@ func TestRemoteWriteAuthorizationConfig(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "sigv4 authentication configuration",
+			config: `prometheusK8s:
+  remoteWrite:
+  - url: "https://authorization.remotewrite.com/api/write"
+    sigv4:
+      region: eu
+      accessKey:
+        name: aws-credentials
+        key: access
+      secretKey:
+        name: aws-credentials
+        key: secret
+      profile: "SomeProfile"
+      roleArn: "SomeRoleArn"
```

Comment:
FYI
RoleArn has minimum filed size of 20, otherwise, would see error in prometheus-k8s pod logs
```
ts=2022-04-14T11:45:44.525Z caller=dedupe.go:112 component=remote level=warn remote_name=a42715 url=https://authorization.remotewrite.com/api/write msg="Failed to send batch, retrying" err="Post \"https://authorization.remotewrite.com/api/write\": failed to sign request: InvalidParameter: 1 validation error(s) found.\n- minimum field size of 20, AssumeRoleInput.RoleArn.\n"
```